packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -82,59 +68,55 @@ export class Breadcrumbs implements Integration {
    *  - Fetch API
    *  - History API
    */
-  public setupOnce(): void {
+  public setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    const hub = getCurrentHub();
```

Comment:
The change to using a local hub vs. the global `getCurrentHub` is more correct, and will help when we switch to passing in the hub/client explicitly.